merge-back.sh

```diff
@@ -2,7 +2,7 @@
 
 test -d .git
 if [ "$?" != "0" ]; then
-    echo "This is no git repository. Exiting..."
+    echo "This is not a git repository. Exiting..."
```

Comment:
Hah! This will make the tests fail in the feature/tests branch, since it's the only test I currently have. Not a problem of course, just noticed it :+1:

Reply:
:wink: